### DISCUSSION TITLE
can you submit same solution as most upvoted after you unlock the solution?

### DISCUSSION THREAD

**Discussion by YGHacker** [codewars/codewars.com#2653] [General]
can you submit same solution as most upvoted, after you find is too difficult and decide to unlock the solution? is it considered cheating? In this case you don't get any honor just for purpose of archive. is it the right way to do?

**Comment by Blind4Basics** [comment]
yes, you can, because you won't get any reward for it.

**Comment by Blind4Basics** [comment]
additional note: though it is to be noted that, 99% of the time, names grouped with the most upvoted solutions are copycats/cheaters. So nothing will happen to you, but you will be seen by other users as a potential copycat (because a lambda user cannot see if you got points or not for that solution)

**Comment by hobovsky** [comment]
What about creating a fork of a solution we are interested with? Wouldnt forks be preferred to submitting a copy?

I didnt use the solution forks, so i don;t know, maybe they have some disadvantages, or work somehow differently?

**Reply by Blind4Basics** [reply]
I guess the actual idea is to get rid of the kata in the "unfinished kata" list. ;)

(this list is creating more problems than anything, imo...)

**Reply by YGHacker** [reply]
Yes actually, to submit is to let myself know that I've tried this problem so that I will not want to see it again.

Also sometimes I cannot even understand the problem, the description or output is not clear.. 

So I just unlock the solution, and want to try to learn the solution and then submit their solution. Maybe there are not so many cheater in top voted question. If its fine then ok.

**Reply by hobovsky** [reply]
Its true that some problems are difficult to understand or confusing, but unlocking solutions is not only possibility, and can be considered wasted points :)
You can always leave such kata for later and come back to it after some time, or ask for help on Codewars Discord.
